### PR TITLE
Fix multimode interface double box on file upload

### PR DESCRIPTION
**Pull request by TALLEC-Scott**
## Description

When file is provided but string in input is empty it still gets added to message history which renders empty string and generates a senseless box. See issue for example on this.

Closes: #8013

### CODE REVIEW

File: gradio/chat_interface.py [hunk 1]
```diff
@@ -202,7 +202,10 @@ def __init__(
             with Group():
                 with Row():
                     if textbox:
-                        textbox.container = False
+                        if self.multimodal:
```
Comment (freddyaboulton) [inline gradio/chat_interface.py:205]: This makes the textbox look squished in the multimodal case

image

I think we just need `submit_btn=None` in this case

Comment (TALLEC-Scott) [reply gradio/chat_interface.py:205]: That's strange, I flipped the lines. Good catch. Should be good now.

**Review by TALLEC-Scott — COMMENTED** [review]
Check  my newest commit

**Review by freddyaboulton — APPROVED** [review]
Thanks @TALLEC-Scott !

**Review by dawoodkhan82 — APPROVED** [review]
lgtm! Thanks for the fix @TALLEC-Scott

**Pull request merged**